### PR TITLE
Add ericsson

### LINKED ISSUE #5915: Ericsson
**Brand Name:** Ericsson
**Website:** https://www.ericsson.com/
**Alexa rank:** 14,037
**Official resources for icon and color:** `#0c0c0c` (black) - taken in the favicon, or `#f9d32f` (yellow)? (Also See Below in the Ericsson homepage.)

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
ericsson

**Issue:** closes #5915 

**Similarweb rank:** 41,230. See https://www.similarweb.com/website/ericsson.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from brand resource kit available [here](https://mediabank.ericsson.net/admin/mb/?h=dbeb87a1bcb16fa379c0020bdf713872) in the section Ericsson Logotype and Econ. I took this file `171115_ericsson_logo\_Digital\Econ\Black\SVG\ECON_RGB.svg`
Color: Taken from logo which is also mentioned in [Ericsson Brand Guidelines - Extract from Ericsson Brand House v.2.0](https://mediabank.ericsson.net/admin/mb/?h=dbeb87a1bcb16fa379c0020bdf713872) as to be `#000000`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3820,6 +3820,12 @@
             "hex": "ED2224",
             "source": "https://metal.equinix.com"
         },
+        {
+            "title": "Ericsson",
+            "hex": "000000",
```
Comment (adamrusted) [inline _data/simple-icons.json:3825]: The below is described as 'Ericsson Blue'.

```suggestion
            "hex": "0082F00",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Just the change to the hex (guidelines state the icon should only be shown in black on a light background, or white on a darker background, but this is the brand colour) and this is good to go. Icon is a perfect match.

**Comment by PratyushJayachandran** [comment]
> Just the change to the hex (guidelines state the icon should only be shown in black on a light background, or white on a darker background, but this is the brand colour) and this is good to go. Icon is a perfect match.

Ya I was confused about this. I committed your suggestion.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -3820,6 +3820,12 @@
             "hex": "ED2224",
             "source": "https://metal.equinix.com"
         },
+        {
+            "title": "Ericsson",
+            "hex": "0082f0",
```
Comment (adamrusted) [inline _data/simple-icons.json:3825]:
```suggestion
            "hex": "0082F0",
```

Needs to be uppercase to keep the linter happy, apologies for the additional 0 in my original review...

**Pull request merged**